From 4b9fa38e0d1fc7b02c2ec732d1002ba5e7b33a2d Mon Sep 17 00:00:00 2001
From: Jingoo Han <jg1.han@samsung.com>
Date: Tue, 30 Jul 2013 16:58:59 +0900
Subject: spi: use dev_get_platdata()

Use the wrapper function for retrieving the platform data instead of
accessing dev->platform_data directly.

Signed-off-by: Jingoo Han <jg1.han@samsung.com>
Signed-off-by: Mark Brown <broonie@linaro.org>
(cherry picked from commit 8074cf063e410a2c0cf1704c3b31002e21f5df7c)
Signed-off-by: Simon Horman <horms+renesas@verge.net.au>

Conflicts:
	drivers/spi/spi-altera.c
	drivers/spi/spi-ath79.c
	drivers/spi/spi-au1550.c
	drivers/spi/spi-bcm63xx.c
	drivers/spi/spi-bfin-sport.c
	drivers/spi/spi-bfin-v3.c
	drivers/spi/spi-bfin5xx.c
	drivers/spi/spi-coldfire-qspi.c
	drivers/spi/spi-davinci.c
	drivers/spi/spi-ep93xx.c
	drivers/spi/spi-fsl-espi.c
	drivers/spi/spi-fsl-lib.c
	drivers/spi/spi-fsl-spi.c
	drivers/spi/spi-gpio.c
	drivers/spi/spi-mpc512x-psc.c
	drivers/spi/spi-mpc52xx-psc.c
	drivers/spi/spi-nuc900.c
	drivers/spi/spi-oc-tiny.c
	drivers/spi/spi-omap-100k.c
	drivers/spi/spi-omap2-mcspi.c
	drivers/spi/spi-pl022.c
	drivers/spi/spi-rspi.c
	drivers/spi/spi-s3c24xx.c
	drivers/spi/spi-s3c64xx.c
	drivers/spi/spi-sh-sci.c
	drivers/spi/spi-ti-ssp.c
	drivers/spi/spi-tle62x0.c
	drivers/spi/spi-xilinx.c
---
 drivers/spi/spi-sh-msiof.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/spi/spi-sh-msiof.c b/drivers/spi/spi-sh-msiof.c
index 6688ce78df78..2a95435a6a11 100644
--- a/drivers/spi/spi-sh-msiof.c
+++ b/drivers/spi/spi-sh-msiof.c
@@ -645,7 +645,7 @@ static int sh_msiof_spi_probe(struct platform_device *pdev)
 	if (pdev->dev.of_node)
 		p->info = sh_msiof_spi_parse_dt(&pdev->dev);
 	else
-		p->info = pdev->dev.platform_data;
+		p->info = dev_get_platdata(&pdev->dev);
 
 	if (!p->info) {
 		dev_err(&pdev->dev, "failed to obtain device info\n");
-- 
1.8.5.rc3

